reana_client/cli/workflow.py

```diff
@@ -81,9 +81,6 @@ def workflow_execution_group(ctx):
     default=None,
     help="Get output in JSON format.",
 )
-@click.option(
```

Comment:
Please keep it for backward compatibility, just update docstring that it is deprecated, that deleted worklows are now listed by default, etc.

Reply:
ohh okay, I was the reading the issue again. I thought I missed something 😅